addons/survey/wizard/survey_invite.py

```diff
@@ -217,6 +217,9 @@ def _get_answers_values(self):
 
     def _send_mail(self, answer):
         """ Create mail specific for recipient containing notably its access token """
+        email_from = self.template_id._render_field('email_from', answer.ids)[answer.id] if self.template_id.email_from else self.author_id.email_formatted
+        if not email_from:
```

Comment:
Note that it is not necessary, as normally it crashes when creating the underlying mail.message. And it avoid updating translations (maybe forgot to remove it previously, but let us update it here).